pkg/msvc-uwp/RetroArch-msvc2017-UWP/RetroArch-msvc2017-UWP.vcxproj

```diff
@@ -132,7 +132,7 @@
     <PackageCertificateKeyFile>RetroArch-UWP_TemporaryKey.pfx</PackageCertificateKeyFile>
     <GenerateAppInstallerFile>False</GenerateAppInstallerFile>
     <AppxAutoIncrementPackageRevision>False</AppxAutoIncrementPackageRevision>
-    <AppxBundlePlatforms>x86|x64|arm</AppxBundlePlatforms>

```

Comment:
The only issue I have with this PR is that x86|x64|arm were all removed. Can you add them back?

Reply:
OK, fixed now.